nuimo/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(name='nuimo',
+      version='1.0.0',
+      description='Nuimo SDK for Python on Linux',
+      url='https://github.com/getsenic/nuimo-linux-python',
+      maintainer='Lars Blumberg',
+      maintainer_email='lars@senic.com',
+      license='MIT',
+      install_requires=['gattlib'],
```

Comment:
Are you sure this will work? I did try in the past but this was not Python v3 compatible. I guess the right lib is called something like gattlib3 or something else